app/src/main/java/com/android/solvit/MainActivity.kt

```diff
@@ -115,6 +116,7 @@ fun SeekerUI(
     composable(Route.ORDER) {
       EditRequestScreen(navigationActions)
     } // This line can be replace when the OrderScreen is implemented
+    composable(Screen.CALENDAR) { ProviderCalendarScreen(navigationActions = navigationActions) }
```

Comment:
You have to pull latest main version so that you can merge without conflicts as whole main activity file has changed